stiebeleltron/integrationpluginstiebeleltron.json

```diff
@@ -76,30 +76,249 @@
+                            "displayNameEvent": "Cooling mode status changed",
+                            "type": "bool",
+                            "defaultValue": false,
+                            "suggestLogging": true
+                        },
+                        {
+                            "id": "5c47d9bb-66de-48aa-b90e-caa3ca8d44a5",
+                            "name": "defrosting",
+                            "displayName": "Defrosting mode",
+                            "displayNameEvent": "Defrosting mode status changed",
+                            "type": "bool",
+                            "defaultValue": false,
+                            "suggestLogging": true
+                        },
+                        {
+                            "id": "047fcb7d-9080-4b45-9a6e-5060fa43f7c2",
+                            "name": "silentMode",
+                            "displayName": "Silent mode",
+                            "displayNameEvent": "Silent mode status changed",
+                            "type": "bool",
+                            "defaultValue": false,
+                            "suggestLogging": true
+                        },
+                        {
+                            "id": "d77a30d9-98f7-40ec-bc55-77c547f24145",
+                            "name": "silentMode2",
+                            "displayName": "Silent mode 2 (Off)",
+                            "displayNameEvent": "Silent mode 2 status changed",
+                            "type": "bool",
+                            "defaultValue": false,
+                            "suggestLogging": true
+                        },     
+                        {
+                            "id": "798458bb-d38e-4028-880a-6dcba409a2f5",
+                            "name": "sgReadyActive",
+                            "displayName": "SG Ready enabled",
+                            "displayNameEvent": "SG Ready activation changed",
+                            "displayNameAction": "Switch SG Ready on/off",
+                            "type": "bool",
+                            "writable": true,
+                            "defaultValue": false,
+                            "suggestLogging": true
+                        },
                         {
                             "id": "7d474fb5-aa37-4f21-8166-b20f5bf84fb4",
                             "name": "sgReadyMode",
-                            "displayName": "Smart grid mode",
-                            "displayNameEvent": "Smart grid mode changed",
-                            "displayNameAction": "Set smart grid mode",
+                            "displayName": "SG Ready mode",
+                            "displayNameEvent": "SG Ready mode changed",
+                            "displayNameAction": "Set SG Ready mode",
                             "type": "QString",
                             "possibleValues": [
-                                "Off",
```

Comment:
Please note that the values for the SG ready state need to match the interface. 
This is actually a bug in the plugininfo compiler that you where able to build it like this.

https://preview.nymea.io/documentation/resources/interfaces#smartgridheatpump

Reply:
See https://github.com/nymea/nymea-plugins-modbus/pull/55/commits/5f029352bd1812deb69b9f9d5e2edd956ad9c1dd